src/styles/blocks/header.scss

```diff
@@ -1,3 +1,5 @@
+$header-transition-duration: 0.3s;
```

Comment:
The transition duration should be the same throughout the page. So create a file for variables and put the `transition-duration` there